src/moneywiz_api/model/transaction.py

```diff
@@ -434,12 +486,15 @@ def validate(self):
 
         assert self.original_exchange_rate is not None
 
-        assert self.amount == self.original_amount
-        assert self.amount == pytest.approx(
-            -self.recipient_amount / self.original_exchange_rate,
-            abs=0.001,
+        # assert self.amount == self.original_amount
```

Comment:
any reason?

Reply:
For Crypto/FX accounts, the amount shows a value in the account currency, such as USD. The original amount could be BTC, ETC, etc.

Reply:
The assertion needs to depend on account type, I suppose